frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { match as RouterMatch } from 'react-router';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { Alert, Button } from '@patternfly/react-core';
+import { StorageClassResourceKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { history } from '@console/internal/components/utils';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { fetchK8s } from '@console/internal/graphql/client';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager';
+import { LSO_NAMESPACE } from '@console/local-storage-operator-plugin/src/constants';
+import { CreateOCS } from './install-lso-sc';
+import { scResource, LSOResource } from '../../../constants/resources';
+import { filterNoProvSC } from '../../../utils/install';
+
+import './attached-devices.scss';
+
+export const CreateOCSOnAttachedDevices: React.FC<CreateOCSOnAttachedDevicesProps> = ({
+  match,
+}) => {
+  const [isNoProvSCPresent, SetIsNoProvSCPresent] = React.useState(false);
+  // LSO stands for local-storage-operator
+  const [LSOEnabled, setLSOEnabled] = React.useState(false);
+  const [scData, scLoaded, scLoadError] = useK8sWatchResource<StorageClassResourceKind[]>(
+    scResource,
+  );
+  const [LSOData, LSOLoaded, LSOLoadError] = useK8sWatchResource<K8sResourceKind>(LSOResource);
+
+  React.useEffect(() => {
+    if ((LSOLoadError || !LSOData) && LSOLoaded) {
+      setLSOEnabled(false);
+    } else if (LSOLoaded) {
+      fetchK8s(ClusterServiceVersionModel, LSOData?.status?.currentCSV, LSO_NAMESPACE)
+        .then(() => {
+          setLSOEnabled(true);
+        })
+        .catch(() => {
+          setLSOEnabled(false);
+        });
+    }
+  }, [LSOData, LSOLoaded, LSOLoadError]);
+
+  React.useEffect(() => {
+    if ((scLoadError || scData.length === 0) && scLoaded) {
+      SetIsNoProvSCPresent(false);
+    } else if (scLoaded) {
+      const filteredSCData = scData.filter(filterNoProvSC);
+      if (filteredSCData.length) {
+        SetIsNoProvSCPresent(true);
+      }
+    }
+  }, [scData, scLoaded, scLoadError]);
+
+  const takeLSOInstallationPage = () => {
+    history.push(
+      '/operatorhub/all-namespaces?details-item=local-storage-operator-redhat-operators-openshift-marketplace',
+    );
+  };
+
+  return (
+    <div className="co-m-pane__body">
+      {!LSOEnabled && (
+        <Alert
+          className="co-alert"
+          variant="info"
+          title="Local Storage Operator Not Installed"
+          isInline
+        >
+          <div>
+            Before we can create a storage cluster, the local storage operator needs to be
+            installed. When installation is finished come back to OpenShift Container Storage to
+            create a storage cluster.
+            <div className="ceph-ocs-install__lso-alert">
```

Comment:
It does not looks like following BEM.
The element is a button here.
```suggestion
            <div className="ceph-ocs-install-alert__button">
```